<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BaseInfo</class>
 <widget class="QWidget" name="BaseInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>设置基本信息</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>231</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>案件基本信息</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>21</y>
      <width>62</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>案件编号：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>  立卷人：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>  发文号：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>森公单位：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>林业单位：</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>21</y>
      <width>141</width>
      <height>121</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QLineEdit" name="caseNumberEdit"/>
    <widget class="QLineEdit" name="caseCopEdit"/>
    <widget class="QWidget" name="layoutWidget">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="caseFaWenNum1"/>
      </item>
      <item>
       <widget class="QLineEdit" name="caseFaWenNum2"/>
      </item>
     </layout>
    </widget>
    <widget class="QLineEdit" name="caseForestUnitEdit"/>
    <widget class="QLineEdit" name="caseLinYeUnitEdit"/>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>291</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>当事人基本信息</string>
   </property>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>261</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>民族：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="caseNationEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>文化：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="caseCultureEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>职业：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="caseJobEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>261</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>身份证号码：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="caseHumNumberEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>251</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>家庭住址：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="caseHomeEdit"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>510</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确  定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>510</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取 消</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>291</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>CSI</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>271</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>勘验时间：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="csiTimeEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>271</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>勘验地点：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="csiLocEdit"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>291</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>其他</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>271</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>办案人员：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="casePolEdit1"/>
     </item>
     <item>
      <widget class="QLineEdit" name="casePolEdit2"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>271</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>复议机构：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="caseFuYiEdit"/>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>caseNumberEdit</tabstop>
  <tabstop>caseCopEdit</tabstop>
  <tabstop>caseFaWenNum1</tabstop>
  <tabstop>caseFaWenNum2</tabstop>
  <tabstop>caseForestUnitEdit</tabstop>
  <tabstop>caseLinYeUnitEdit</tabstop>
  <tabstop>caseNationEdit</tabstop>
  <tabstop>caseCultureEdit</tabstop>
  <tabstop>caseJobEdit</tabstop>
  <tabstop>caseHumNumberEdit</tabstop>
  <tabstop>caseHomeEdit</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>pushButton_2</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>BaseInfo</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>252</x>
     <y>302</y>
    </hint>
    <hint type="destinationlabel">
     <x>290</x>
     <y>343</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
